.changeset/pretty-eels-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@axis-backstage/plugin-jira-dashboard': minor
+---
+
+Introduce Pagination to jira table for better visualization
```

Comment:
Let's add that something like 

"tableOptions" has been added as props to the EntityJiraDashboardContent component. This makes it possible to configure table in more ways.